velox/core/QueryConfig.h

```diff
@@ -286,7 +286,7 @@ class QueryConfig {
   /// deleted when no longer needed. Files may be left behind after crashes but
   /// are identifiable based on the Task id in the name.
   std::optional<std::string> spillPath() const {
-    return get<std::string>(kSpillPath, "/tmp");
+    return get<std::string>(kSpillPath, "");
```

Comment:
The 'get' method with default value parameter never returns a T, not an optional. Hence, this method always returns an option that's set. It looks like this is an oversights and the method should either return T or we should not pass default value to 'get'.

Reply:
Hmm, I don't think I understand this (some grammar issue?):

> The 'get' method with default value parameter never returns a T, not an optional.

But I can remove the default value, which is an empty string. Although, as the method will be removed in the following changes, this is not important.